mhs/common/mhs_common/workflow/tests/test_asynchronous_reliable.py

```diff
@@ -18,6 +18,7 @@
 from mhs_common.messages import ebxml_request_envelope, ebxml_envelope
 from mhs_common.state import work_description
 from mhs_common.state.work_description import MessageStatus
+from json import loads as load_json
```

Comment:
just `import json` and use `json.loads`. `load/loads` and `dump/dumps` are standard names also used by `marshall` and `pickle`